backend/Node/routes/route-helpers/spotify-helpers.ts

```diff
@@ -20,3 +23,92 @@ export const scopes = [
 	'user-follow-read',
 	'user-follow-modify',
 ];
+
+export const createSpotifyWebApi = () => {
+	return new SpotifyWebApi({
+		clientId: process.env.SPOTIFY_CLIENT_ID,
+		clientSecret: process.env.SPOTIFY_CLIENT_SECRET,
+		redirectUri: 'http://localhost:4000/spotify/callback',
+	});
+};
+
+export async function getPopularityForTracks(req: any, res: any) {
+	var spotifyApi = createSpotifyWebApi();
```

Comment:
can we replace `var` with `let` here (and everywhere else)

Reply:
done